core/src/main/resources/jenkins/security/UpdateSiteWarningsConfiguration/config.properties

```
@@ -20,10 +20,10 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-warning.core = Jenkins core: {0}
-warning.plugin = {0}: {1} 
+warning.core=Jenkins core: {0}
+warning.plugin={0}: {1} 
 
 blurb = This section allows you to suppress warnings, applicable to your Jenkins \
-  configuration, provided by the updates sites. If you do, they won’t be shown by \
+  configuration, provided by the update site. If you do, they won’t be shown by \
```

there may be one or more update sites, Jenkins on its own has only 1 but there's a plugin that allows you to add more and I think CloudBees may use multiple,?
```suggestion
  configuration, provided by the update site(s). If you do, they won’t be shown by \
```